firebase-installations/src/test/java/com/google/firebase/installations/FirebaseInstallationsTest.java

```diff
@@ -140,7 +144,10 @@ public class FirebaseInstallationsTest {
   public void setUp() {
     MockitoAnnotations.initMocks(this);
     FirebaseApp.clearInstancesForTest();
-    executor = new ThreadPoolExecutor(0, 1, 30L, TimeUnit.SECONDS, new LinkedBlockingQueue<>());
+    backgroundExecutor = TestOnlyExecutors.background();
+    networkExecutor =
+        FirebaseExecutors.newSequentialExecutor(
+            new ThreadPoolExecutor(0, 5, 30L, TimeUnit.SECONDS, new LinkedBlockingQueue<>()));
```

Comment:
Use the blocking executor from TestOnlyExecutors here too. The interface they implement is https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ScheduledExecutorService.html which implements both `Executor` and `ExecutorService`.